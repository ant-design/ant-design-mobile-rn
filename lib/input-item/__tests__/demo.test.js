import rnDemoTest from '../../../tests/shared/demoTest';

rnDemoTest('input-item');
