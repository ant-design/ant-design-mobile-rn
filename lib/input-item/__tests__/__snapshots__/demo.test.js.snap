// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/input-item/demo/basic.tsx correctly 1`] = `
<RCTScrollView
  automaticallyAdjustContentInsets={false}
  keyboardShouldPersistTaps="handled"
  showsHorizontalScrollIndicator={false}
  showsVerticalScrollIndicator={false}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View>
    <View>
      <View>
        <Text
          style={
            Object {
              "backgroundColor": "#f5f5f9",
              "color": "#888888",
              "fontSize": 14,
              "paddingBottom": 9,
              "paddingHorizontal": 15,
              "paddingTop": 15,
            }
          }
        >
          基本
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#ffffff",
            "borderTopColor": "#dddddd",
            "borderTopWidth": 0.5,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            输入框
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="有标签"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                Object {
                  "color": "#f50",
                },
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
          <View
            accessible={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#888888",
                    "fontSize": 15,
                    "marginLeft": 5,
                  },
                  Object {
                    "width": 17,
                  },
                ]
              }
            >
              元
            </Text>
          </View>
          <View
            accessible={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Array [
                Object {
                  "height": 21,
                  "marginLeft": 5,
                  "width": 21,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#cccccc",
                    "fontFamily": "antoutline",
                    "fontSize": 22,
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {
                    "color": "#f4333c",
                  },
                ]
              }
            >
              
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            输入框
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={false}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="不可编辑"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value="不可编辑"
          />
          <View
            accessible={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
          >
            <Text>
              元
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            输入框
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={false}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="disabled"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {
                  "backgroundColor": "#dddddd",
                  "color": "#bbbbbb",
                },
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value="disabled"
          />
          <View
            accessible={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
          >
            <Text>
              元
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="无标签"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            标题
          </Text>
          <TextInput
            allowFontScaling={true}
            autoFocus={false}
            clearButtonMode="while-editing"
            defaultValue="xx"
            editable={true}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="自动获取光标"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
          />
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            标题
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="点击下方按钮该输入框会获取光标"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
          />
        </View>
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={null}
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#ffffff",
                  "flexDirection": "row",
                  "flexGrow": 1,
                  "paddingLeft": 15,
                },
                undefined,
              ]
            }
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "borderBottomColor": "#dddddd",
                    "borderBottomWidth": 0.5,
                    "flex": 1,
                    "flexDirection": "row",
                    "minHeight": 44,
                    "paddingRight": 15,
                    "paddingVertical": 6,
                  },
                  false,
                  false,
                ]
              }
            >
              <View
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "flexDirection": "column",
                    },
                  ]
                }
              >
                <View
                  accessible={true}
                  isTVSelectable={true}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "borderRadius": 5,
                        "borderWidth": 1,
                        "justifyContent": "center",
                      },
                      Object {
                        "height": 47,
                        "paddingLeft": 15,
                        "paddingRight": 15,
                      },
                      Object {
                        "backgroundColor": "#108ee9",
                        "borderColor": "#108ee9",
                      },
                      false,
                      false,
                      undefined,
                      undefined,
                    ]
                  }
                >
                  <View
                    style={
                      Object {
                        "flexDirection": "row",
                      }
                    }
                  >
                    <Text
                      style={
                        Array [
                          Object {
                            "fontSize": 18,
                          },
                          Object {
                            "color": "#ffffff",
                          },
                          false,
                          false,
                        ]
                      }
                    >
                      点击获取光标
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#ffffff",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "bottom": 0,
                "height": 1,
                "left": 0,
                "position": "absolute",
                "right": 0,
              },
            ]
          }
        />
      </View>
    </View>
    <View>
      <View>
        <Text
          style={
            Object {
              "backgroundColor": "#f5f5f9",
              "color": "#888888",
              "fontSize": 14,
              "paddingBottom": 9,
              "paddingHorizontal": 15,
              "paddingTop": 15,
            }
          }
        >
          固定标签字数
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#ffffff",
            "borderTopColor": "#dddddd",
            "borderTopWidth": 0.5,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 35.7,
                },
              ]
            }
          >
            姓名
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="两个字标签"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 53.550000000000004,
                },
              ]
            }
          >
            校验码
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="三个字标签"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            四字标签
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="四个字标签（默认）"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#ffffff",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "bottom": 0,
                "height": 1,
                "left": 0,
                "position": "absolute",
                "right": 0,
              },
            ]
          }
        />
      </View>
    </View>
    <View>
      <View>
        <Text
          style={
            Object {
              "backgroundColor": "#f5f5f9",
              "color": "#888888",
              "fontSize": 14,
              "paddingBottom": 9,
              "paddingHorizontal": 15,
              "paddingTop": 15,
            }
          }
        >
          格式
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#ffffff",
            "borderTopColor": "#dddddd",
            "borderTopWidth": 0.5,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            文本输入
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="text"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                Object {
                  "color": "#f50",
                },
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
          <View
            accessible={true}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Array [
                Object {
                  "height": 21,
                  "marginLeft": 5,
                  "width": 21,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#cccccc",
                    "fontFamily": "antoutline",
                    "fontSize": 22,
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {
                    "color": "#f4333c",
                  },
                ]
              }
            >
              
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            银行卡
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="number-pad"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="bankCard"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            手机号
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="phone-pad"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="phone"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            密码
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="default"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="password"
            secureTextEntry={true}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "flexDirection": "row",
                "height": 44.5,
                "marginBottom": 0,
                "marginLeft": 15,
                "marginTop": 0,
                "paddingRight": 15,
              },
              Object {
                "borderBottomWidth": 0.5,
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontSize": 17,
                  "marginRight": 5,
                  "textAlignVertical": "center",
                },
                Object {
                  "width": 71.4,
                },
              ]
            }
          >
            数字
          </Text>
          <TextInput
            allowFontScaling={true}
            clearButtonMode="while-editing"
            editable={true}
            keyboardType="numeric"
            labelPosition="left"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            placeholder="number"
            secureTextEntry={false}
            style={
              Array [
                Object {
                  "height": 35,
                },
                Object {
                  "backgroundColor": "transparent",
                  "color": "#000000",
                  "flex": 1,
                  "fontSize": 17,
                },
                null,
                Object {},
                undefined,
              ]
            }
            textAlign="left"
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#ffffff",
                "borderBottomColor": "#dddddd",
                "borderBottomWidth": 0.5,
                "bottom": 0,
                "height": 1,
                "left": 0,
                "position": "absolute",
                "right": 0,
              },
            ]
          }
        />
      </View>
    </View>
  </View>
</RCTScrollView>
`;
